javascripts/map.js

```diff
@@ -1519,8 +1519,11 @@ function printDirections(){
 
 //determine which stylesheet to use, mobile or desktop
 function isMobile() {
-  try{ document.createEvent("TouchEvent"); return true; }
-  catch(e){ return false; }
+	if (navigator.userAgent.match(/Mobi/))
```

Comment:
Agreed, thinking about it after the fact. and maintaining a list would get awful, awful fast...
`alcatel|amoi|android|avantgo|blackberry|benq|cell|cricket|docomo|elaine|htc|iemobile|iphone|ipad|ipaq|ipod|j2me|java|midp|mini|mmp|mobi|motorola|nec-|nokia|palm|panasonic|philips|phone|sagem|sharp|sie-|smartphone|sony|symbian|t-mobile|telus|up\.browser|up\.link|vodafone|wap|webos|wireless|xda|xoom|zte`